samples/Tests/ColumnSet.Input.Text.VerticalStretch.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Make sure you rerun the test app and add these files if they show up as new again. With my serialization change you need to add the *ToJson.json files to server as a baseline for the json roundtripping. Clicking Set As Expected should do this for you.